addons/product/models/product_pricelist.py

```diff
@@ -155,7 +155,7 @@ def _compute_price_rule(self, products_qty_partner, date=False, uom_id=False):
             (prod_tmpl_ids, prod_ids, categ_ids, self.id, date, date))
 
         item_ids = [x[0] for x in self._cr.fetchall()]
-        items = self.env['product.pricelist.item'].browse(item_ids)
+        items = self.env['product.pricelist.item'].search([('id', 'in', item_ids)])
```

Comment:
add `with_context(active_test=False)`

Reply:
Does it make sense to include disabled rules?